psalm.xml.dist

```diff
@@ -43,6 +43,7 @@
                 <referencedClass name="Doctrine\ORM\ORMException"/>
                 <!-- Dropped in DBAL 4 -->
                 <referencedClass name="Doctrine\DBAL\Exception"/>
+                <referencedClass name="Symfony\Bridge\Doctrine\Middleware\IdleConnection\Driver"/>
```

Comment:
Add a comment like was suggested please. It should include symfony version where class is available